src/Microsoft.SqlTools.ServiceLayer/TableDesigner/DesignerPathUtils.cs

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections;
+using System.Linq;
+using Microsoft.SqlTools.ServiceLayer.TableDesigner.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.TableDesigner
+{
+    public static class DesignerPathUtils
+    {
+        ///<summary>
+        /// Parse the path in the table designer change information.
```

Comment:
Please given information about what the format of this path is supposed to be like.